charts/kueue/README.md

```diff
@@ -48,6 +48,46 @@ NAME                           READY   UP-TO-DATE   AVAILABLE   AGE
 kueue-controller-manager       1/1     1            1           7s
 ```
 
+##### Cert Manager
+
+Kueue has support for third-party certificates.
+One can enable this by setting `enableCertManager` to true.
+This will use certManager to generate a secret, inject the CABundles and set up the tls.
+
+##### Protecting metrics endpoints with certificates
+
+Assuming one has set up cert manager and prometheus, Kueue's helm chart provides
+support for third-party certificates for metrics also.
+
+One can enable this by setting both `enablePrometheus` and `enableCertManager` to true.
+If one wants to require TLS certificates for metrics endpoints, one needs to provide a tlsConfig to the serviceMonitor.
+
+An example values file could be as follows:
+
+```yaml
+...
+metrics:
+  prometheusNamespace: monitoring
+# tls configs for serviceMonitor
+  serviceMonitor:
+    tlsConfig:
+      serverName: kueue-controller-manager-metrics-service.kueue-system.svc
+      insecureSkipVerify: false
```

Comment:
`insecureSkipVerify: false` is the default. This line could be deleted.

Reply:
I actually wanted to drop this entirely.